README.md

```diff
@@ -24,52 +22,72 @@ To get a development environment running on your local machine:
 cp .env.example .env
 ```
 
-3. Install packages
+### Install packages
 
 ```bash
 npm install
 ```
 
-4. Database Setup
-   You'll need to create a free database with [Xata](https://xata.io/) which runs on Postgres. When you create your db, make sure enable the checkbox that says `Enable direct access to Postgres`.
+### Database Setup
+
+To get started, you'll need to create a **free** account with [Xata](https://xata.io/).
+
+Xata runs on Postgres, but to get access to the Postgres layer, youll need to enable this in your workspace settings. Find your workspace settings and toggle to enable `Direct Access to Postgres (BETA)`.
+
+Now, you'll need to create a new database. Give it a name, and make sure to enable the checkbox that says `Enable direct access to Postgres`.
+
+> In this project, we'll be connecting to our db in two different ways: using Prisma ORM and the Xata client SDK. All standard CRUD (Create, read, update, and delete) db interactions will be run through Prisma. We'll only be using the Xata client to take advantage of Xata specific features like search and file uploads.
+
+Then, select your newly created database. On the next page, you'll want to copy the `PostgreSQL endpoint` property. It will look something like this.
+
+`postgresql://l5kbra:<YOUR_API_KEY>@us-east-1.sql.xata.sh/<DB_NAME>:main?sslmode=require`
```

Comment:
After this I would add:

Next, click the "generate API key and keep note of it. You'll need it in your `.env` file."

Or something along those lines.

Reply:
I'm struggling now because I'm realizing the creation of a first db is different from your "not first" db lol So, this are options I'm not seeing

Reply:
Yup, I saw that you reached out to Xata. It's interesting to me they don't follow the same workflow.